src/features/iam/tree/iamExplorerProvider.ts

```diff
@@ -38,6 +40,14 @@ export class IamExplorerProvider implements ExplorerProvider {
 
   private async getProjects() {
     const projects = await iamState.listProjects()
+    const currentUserId = configVaultService.getCurrentUserID()
+    const activeProjectId = currentUserId
+      ? configVaultService.getActiveProjectId(currentUserId)
+      : null
+
+    if (projects) {
+      setActiveProject(activeProjectId ?? projects[0]?.projectId)
+    }
```

Comment:
Actually did this on IamExplorerProvider because of the acceptance criteria in the jira ticket. 

> There should always be an active project. When loading projects for the first time check config.json to see if the user has a previously set active project. If not then pick the first one and update config files like previously stated.